openpype/settings/entities/schemas/projects_schema/schema_project_houdini.json

```diff
@@ -19,6 +19,39 @@
                 {
                     "type": "template",
                     "name": "template_host_color_management_ocio"
+                },
+                {
+                    "key": "workfile",
+                    "type": "dict",
+                    "label": "Workfile",
+                    "collapsible": true,
+                    "checkbox_key": "enabled",
+                    "children": [
+                        {
+                            "type": "boolean",
+                            "key": "enabled",
+                            "label": "Enabled"
+                        },
+                        {
+                            "type": "label",
+                            "label": "Render space in Houdini is always set to 'scene_linear' Role."
```

Comment:
empty fields (like in my screenshot) fall to default settings from ocio config file where 
- `Display` will be `ACES` (default displays)
- `View` will be `sRGB:DCDM:.........` (default views)
- `Review colorspace` will be the colorspace name for the current `Display` and `View`
![image](https://github.com/ynput/OpenPype/assets/20871534/7c62efff-615a-47c7-89cd-e8e0a011ce7e)

---

so any suggestions on how to rephrase these info in the above help text ? 

```suggestion
                            "label": "Render space in Houdini is always set to 'scene_linear' Role.\nEmpty fields fall to default values retrieved from the ocio config file."
```

Reply:
```suggestion
                            "label": "Empty values will be skipped, allowing any existing env vars to pass through as defined.\nThe render space in Houdini is always set to the 'scene_linear' role."
```

Would this make sense?

> Review colorspace -> the colorspace name for the current default_display and default_view

Can we add placeholder text in this fields when empty? Do text fields in settings support that? That might help clarify what type of data to enter.

Reply:
I think we won't worry about OP as much as Ayon!